packages/squiggle-lang/__tests__/Distributions/Symbolic_test.res

```diff
@@ -0,0 +1,177 @@
+open Jest
+open Expect
+
+let fnImage = (theFn, inps) => Js.Array.map(theFn, inps)
+
+let env: DistributionOperation.env = {
+  sampleCount: 100,
+  xyPointLength: 100,
+}
+
+let mkNormal = (mean, stdev) => GenericDist_Types.Symbolic(#Normal({mean: mean, stdev: stdev}))
```

Comment:
again with making these the right way

Reply:
I'm confused. Getting the dispatch right the _other_ way is a feature we just started talking about today and isn't implemented yet.

Reply:
hmm I actually want to leave it the way it is right now, because we're gonna change the dispatch of validation soon anyway, does that make sense?

Reply:
Yea, fair enough.